test/Emitter/SapiStreamEmitterTest.php

```diff
@@ -58,6 +58,26 @@ public function testEmitCallbackStreamResponse()
         $this->assertSame('it works', ob_get_clean());
     }
 
+    public function testAssertNoPreviousOutput()
+    {
+        $stream = new CallbackStream(function () {
+            return 'it works';
+        });
+        $response = (new Response())
+            ->withStatus(200)
+            ->withBody($stream);
+        ob_start();
+        echo 'Unexpected Output';
+        try {
+            $this->emitter->emit($response);
+        } catch (Exception $e) {
+            $this->assertTrue($e instanceof EmitterException);
+            $this->assertEquals('',$e->getMessage());
+        } finally {
+            ob_get_clean();
+        }
+    }
+    
```

Comment:
Try it like this:
```php
$this->expectException(EmitterException::class);
$this->expectExceptionMessage('Unable to emit response; headers already sent');
$this->expectExceptionMessage(sprintf(
    'output started at %s:%d',
     __FILE__,
     __LINE__ + 3
));
ob_start();
echo 'Unexpected Output';
try {
    $this->emitter->emit($response);
} catch (\Throwable $e) {
    throw $e;
} finally {
    ob_end_clean();
}
```

Reply:
Unfortunately, unsuccessfully ... I assume that sending headers is not called in the SAPI environment of tests, all my attempts at tests are unsuccessful ...